SR2019/2018-10-10-place-confirmation.md

```diff
@@ -0,0 +1,11 @@
+---
+to: Student Robotics 2019
+subject: SR2019 Registration and Kickstart Announcement
+note: This email will be sent multiple times, to those in the list not in the "Contacted" segment
+---
+
+You're in! This email is a confirmation of your teams place at Student Robotics 2019. We are unable to confirm second teams until our deadline of the 26th, due to the space constraints of the venue.
+
+The specific details for the London kickstart will be sent over soon to those who have requested the London venue. The Southampton venue (University of Southampton) will be available to all other teams.
```

Comment:
Maybe mention London is a small venue and thus only go there if you've been confirmed